wiki/Tournaments/CFC/CFIC_1/en.md

```diff
@@ -58,67 +58,101 @@ The Catch French Intermediate Cup is run by various community members by distrib
 
 -----------------------
 
-## Match Schedules: Group Stage
+## Match Schedules: Semifinals
 
-### Friday, 1 March 2019
+### Saturday, 16 March 2019
 
 | Match ID | Player A |  |  | Player B | Match Time (UTC+1) |
 | :-: | --: | :-: | :-: | :-- | :-: |
-| #3 | Aequo3 | ![][flag_FR] | ![][flag_FR] | pierreCHT | **17:00** |
+| #1 | Yruama | ![][flag_FR] | ![][flag_FR] | redjiii | **13:00** |
+| #2 | YANOO123 | ![][flag_FR] | ![][flag_FR] | Realmas | **15:00** |
 
-### Saturday, 2 March 2019
+### Sunday, 17 March 2019
 
 | Match ID | Player A |  |  | Player B | Match Time (UTC+1) |
 | :-: | --: | :-: | :-: | :-- | :-: |
-| #1 | pierreCHT | ![][flag_FR] | ![][flag_FR] | 4L-Chan 3 Porte | **13:00** |
-| #2 | ACinfr | ![][flag_FR] | ![][flag_FR] | Aequo3 | **13:30** |
-| #7 | Yruama | ![][flag_FR] | ![][flag_FR] | oasix42 | **14:00** |
-| #9 | YANOO123 | ![][flag_FR] | ![][flag_FR] | Yruama | **14:45** |
-| #18 | Khalemm | ![][flag_FR] | ![][flag_BE] | -\[Smoked\]- | **15:00** |
-| #10 | oasix42 | ![][flag_FR] | ![][flag_FR] | rpp1 | **15:15** |
-| #6 | 4L-Chan 3 Porte | ![][flag_FR] | ![][flag_FR] | Aequo3 | **15:30** |
-| #11 | rpp1 | ![][flag_FR] | ![][flag_FR] | Yruama | **15:45** |
-| #5 | ACinfr | ![][flag_FR] | ![][flag_FR] | pierreCHT | **16:00** |
-| #13 | floflim | ![][flag_FR] | ![][flag_FR] | Khalemm | **16:00** |
-| #12 | oasix42 | ![][flag_FR] | ![][flag_FR] | YANOO123 | **16:15** |
-| #14 | redjiii | ![][flag_FR] | ![][flag_BE] | -\[Smoked\]- | **16:30** |
-| #16 | Khalemm | ![][flag_FR] | ![][flag_FR] | redjiii | **17:00** |
-| #8 | rpp1 | ![][flag_FR] | ![][flag_FR] | YANOO123 | **17:30** |
-| #4 | 4L-Chan 3 Porte | ![][flag_FR] | ![][flag_FR] | ACinfr | **19:00** |
-
-### Sunday, 3 March 2019
+| #3a¹ | YANOO123 | ![][flag_FR] | ![][flag_FR] | Zard | **13:00** |
+| #3b¹ | Realmas | ![][flag_FR] | ![][flag_FR] | Zard | **13:00** |
+| #3c² | Yruama | ![][flag_FR] | ![][flag_FR] | Khalemm | **15:00** |
+| #3d² | redjiii | ![][flag_FR] | ![][flag_FR] | Khalemm | **15:00** |
 
-| Match ID | Player A |  |  | Player B | Match Time (UTC+1) |
-| :-: | --: | :-: | :-: | :-- | :-: |
-| #19 | Rizhen- | ![][flag_FR] | ![][flag_FR] | Sageru | **13:00** |
-| #21 | Realmas | ![][flag_FR] | ![][flag_FR] | Rizhen- | **13:45** |
-| #24 | Sageru | ![][flag_FR] | ![][flag_FR] | Realmas | **14:30** |
-| #20 | Zard | ![][flag_FR] | ![][flag_FR] | Realmas | **15:00** |
-| #23 | Zard | ![][flag_FR] | ![][flag_FR] | Rizhen- | **16:00** |
-| #15 | -\[Smoked\]- | ![][flag_BE] | ![][flag_FR] | floflim | **17:00** |
-| #17 | redjiii | ![][flag_FR] | ![][flag_FR] | floflim | **17:30** |
-| #22 | Sageru | ![][flag_FR] | ![][flag_FR] | Zard | **18:00** |
+¹ Encounter dependent on loser of Match #2 (SF) and winner of Match #5 (QF)
```

Comment:
two spaces at the end and remove the newline on 96
it's basically a list but indexed by superscript, so...